lib/Cpdf.php

```diff
@@ -76,6 +76,11 @@ class Cpdf
      */
     public $signatureMaxLen = 5000;
 
+    /**
+     * @var bool Is PDF/A compliance mode enabled
+     */
+    public $pdfa = false;
```

Comment:
Do we want to generically support PDF/A since various levels of compliance are defined? Should we just support a specific version which evolves as we update the library.

Also, instead of a flag for PDF/A support should we define constants (e.g., DOMPDF::PDF17, DOMPDF::PDFA1A) for various specifications and store the desired spec in a property, using that to toggle functionality?

Reply:
I think, pdfa is a good start, as it is mandatory 1st of january here.

I tested this solution with  adding "DejaVu Sans" as defaultFont, but verapdf it is still throwing this error:

The font programs for all fonts used for rendering within a conforming file shall be embedded within that file, as defined in ISO 32000-1:2008, 9.9	Failed
1 occurrences	[Hide](https://demo.verapdf.org/#)
PDFont	
Subtype == "Type3" || Subtype == "Type0" || renderingMode == 3 || fontFile_size == 1	
root/document[0]/pages[0](6 0 obj PDPage)/contentStream[0](7 0 obj PDContentStream)/operators[102]/font[0](Helvetica-Bold)

Reply:
Upon further consideration I think we can stick with a simple flag for now. While I like the idea of being able to indicate the level of support using a constant it's unrealistic to expect we'll add varying levels of support anytime soon. We can revisit the implementation in the future if we decide to support other variants of PDF/A.